mscore/scoreview.cpp

```diff
@@ -1112,11 +1117,43 @@ void ScoreView::paint(const QRect& r, QPainter& p)
                   }
             }
 
+
+      // AvsOmr -----
+#ifdef AVSOMR
+      Avs::AvsOmrDrawer omrDrawer;
+      std::shared_ptr<Avs::AvsOmrDrawer::Context> omrDrawCtx = omrDrawer.makeContext(&p, _score);
+      auto pageMeasures = [](Page* page) -> QList<const Measure*> {
+            QList<const Measure*> ml;
+            for (const System* s : page->systems()) {
+                const Measure* lastM = s->lastMeasure();
+                for (const Measure* m = s->firstMeasure(); m; m = m->nextMeasure()) {
```

Comment:
Maybe it would be better to use measures returned by `s->measures()` call (you might need to filter them by checking if a particular `MeasureBase` is a measure with something like `m->isMeasure()`). That call would return exactly those measures which are visible by user which might be important for visual representation in case of multi-measure rests presence.

Reply:
changed to use `s->measures()`